cmd/server/assets/login/login.html

```diff
@@ -73,7 +73,6 @@
           return
         }
 
-        flash("Welcome, " + user.email, "success")
```

Comment:
but the welcome flash gives you feedback while the ajax processes!

Reply:
I found it jarring. Like - I saw a flash of green and then it moved on. Did I miss something? Was I supposed to do something?

Reply:
Fair. Possibly connection-speed dependant. Less is more, lets remove it.

Reply:
Yea, maybe we can do a setTimeout on it? That way it only appears if you are waiting 1s+ or something?

Reply:
Sometimes the redirect is the slow part. The intent was some kind of positive feedback while you wait for that second page.